frontend/src/router.js

```diff
@@ -38,11 +38,19 @@ export default new Router({
       }),
       component: () => import('./views/Question')
     },
+    {
+      path: '/review',
+      name: 'newReview',
+      props: ({query: { cid }}) => ({
+        courseID: String(cid)
+      }),
+      component: () => import('./views/Review')
+    },
     {
       path: '/review/:id',
       name: 'review',
       props: ({params: { id }}) => ({
-        id
+        reviewID: String(id)
```

Comment:
Feel like if we're doing the cast here then it should be impossible for it ever to not be cast?